vignettes/BiostringsQuickOverview.Rmd

```diff
@@ -0,0 +1,221 @@
+---
+title: "Biostrings Quick Overview"
+author:
+- |
+  Hervé Pagès\
+  Fred Hutchinson Cancer Research Center\
+  Seattle, WA
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output:
+  html_document:
+  toc: yes
+  df_print: paged
+  number_sections: yes
+---
+
+Most but not all functions defined in the [Biostrings]{.sans-serif}
+package are summarized here.
+
+::: center
+::: {#table:Low_level_manipulation}
+  **Function**                       **Description**
+  ---------------------------------- ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  `length`                           Return the number of sequences in an object.
+  `names`                            Return the names of the sequences in an object.
+  `[`                                Extract sequences from an object.
+  `head`, `tail`                     Extract the first or last sequences from an object.
+  `rev`                              Reverse the order of the sequences in an object.
+  `c`                                Combine in a single object the sequences from 2 or more objects.
+  `width`, `nchar`                   Return the sizes (i.e. number of letters) of all the sequences in an object.
+  `==`, `!=`                         Element-wise comparison of the sequences in 2 objects.
+  `match`, `%in%`                    Analog to `match` and `%in%` on character vectors.
+  `duplicated`, `unique`             Analog to `duplicated` and `unique` on character vectors.
+  `sort`, `order`                    Analog to `sort` and `order` on character vectors, except that the ordering of DNA or Amino Acid sequences doesn't depend on the locale.
+  `relist`, `split`, `extractList`   Analog to `relist` and `split` on character vectors, except that the result is a *DNAStringSetList* or *AAStringSetList* object. `extractList` is a generalization of `relist` and `split` that supports *arbitrary* groupings.
+
+  : **Low-level manipulation of *DNAStringSet* and *AAStringSet*
+  objects.**
+:::
+:::
+
+::: center
+::: {#table:Counting_tabulating}
++--------------------------------+------------------------------------+
+| **Function**                   | **Description**                    |
++:===============================+:===================================+
+| `alphabetFrequency`            | Tabulate the letters (all the      |
+|                                | letters in the alphabet for        |
+| `letterFrequency`              | `alphabetFrequency`, only the      |
```

Comment:
When using `BiocStyle`, I noticed that the function names on the left side will indent. This can be corrected by removing the space in between them. For example

```
| `alphabetFrequency`            | Tabulate the letters (all the      |
| `letterFrequency`              | letters in the alphabet for        |
|                                | `alphabetFrequency`, only the      |
```